examples/demo/financial/correlations.py

```diff
@@ -197,7 +195,7 @@ def _selections_updated(self, event):
             )
             self.corr_plot.request_redraw()
 
-    @observe("sym1,sym2")
+    @observe("symbols,sym1,sym2")
```

Comment:
this was needed to get the plot to show up immediately when the application launches.  Otherwise a plot would only appear after changing sym1 or sym2.